cmd/swarmctl/node/common.go

```diff
@@ -138,6 +138,16 @@ func getNode(ctx context.Context, c api.ControlClient, input string) (*api.Node,
 		}
 
 		if l := len(rl.Nodes); l > 1 {
+			for _, n := range rl.Nodes {
+				name := n.Spec.Annotations.Name
```

Comment:
I see. The use of `Annotations.Name` in "node ls" seems wrong. It looks like Docker prints the hostname in the list. I think we should update both of these to only use the hostname.